test.py

```diff
@@ -9,18 +9,26 @@
 import tempfile
```

Comment:
If you mean tempfile, it can now be removed. I had initially implemented the output for in temp but later changed to store it in the working dir probably for developer obviousness.